### ISSUE TITLE
9.3+1.1 Gobal Configuration is empty / blank

### ISSUE THREAD

**Issue by aldevar** [fusioninventory/fusioninventory-for-glpi#2605]
After installing fusioninventory (not an upgrade) on glpi 9.3, the 'Global Configuration' page is empty.
No errors are displayed in the apache logs.
![image](https://user-images.githubusercontent.com/18332562/43086157-fbfdf776-8e9c-11e8-9ab5-4f241c287008.png)
I cannot configure the plugin.

**Comment by ddurieux** [comment]
You do have a PHP fatal error in logs (in your webserver error log), we need it ;)

**Comment by aldevar** [comment]
I've just checked for errors in logs again and there is absolutely nothing.

OS is Ubuntu 18.04 up2date
php 7.2.7
Apache 2.4.9

VirtualHost is configured like this : 
`<VirtualHost *:443>
  ServerName glpi.sib.fr
  ServerAlias glpiv2.sib.fr
  ServerAdmin stprod@sib.fr
  ErrorLog /var/log/apache2/glpi.err
  CustomLog /var/log/apache2/glpi.log combined
  DocumentRoot /var/www/glpi93
  php_value session.gc_maxlifetime 36000
  <IfModule headers_module>
        Header set Access-Control-Allow-Origin "*"
        Header set Access-Control-Allow-Methods "GET, HEAD, POST, OPTIONS, PUT, DELETE"
        Header set Access-Control-Allow-Headers "Origin,Content-Type,Accept,Authorization,App-Token,Session-Token"
        Header set Access-Control-Allow-Credentials "true"
 </IfModule>
 <Directory "/var/www/glpi93">
    Order allow,deny
    Allow from all
        AllowOverride all
  </Directory>
        SSLEngine On
        SSLProxyEngine on
        SSLCertificateFile edited
        SSLCertificateKeyFile edited
        SSLCertificateChainFile edited
</VirtualHost>
`

file /var/log/apache2/glpi.err is empty. 
file /var/log/apache2/glpi.log => https://gist.github.com/aldevar/b9c59bf89c6c22b7c3dadb90fa42d4dc

**Comment by ddurieux** [comment]
really strange, perhaps in browser console / network (dev tools) you will have more info about an error

**Comment by aldevar** [comment]
I've just succeeded to resolve the issue. Here is the explanation : 
The SQL files in plugins/fusioninventory/install/mysql creates the tables using the MyISAM engine. I'm running a mariadb Galera cluster where tables must be in the InnoDB engine, otherwise they can't be replicated over the cluster.
So, I edited the files in plugins/fusioninventory/install/mysql with a search and replace : 
's/ENGINE=MyISAM/ENGINE=InnoDB/g'
Also, not every table creation is done in the same way, so, I had to run an other search and replace :
's/ENGINE = MyISAM/ENGINE=InnoDB/g'

I then uninstalled the plugin, drop all the fusioninventory tables and reinstalled the plugin.
Now everything works fine.
